packages/web-components/src/components/va-button/va-button.scss

```diff
@@ -14,6 +14,12 @@
 :host([disabled]:not([disabled='false'])) {
   cursor: not-allowed;
 }
+
+// Override transparent background from USWDS v3
```

Comment:
@humancompanion-usds requested to override transparent background from USWDS v3 secondary buttons to white